.gitignore

```diff
@@ -1,2 +1,3 @@
 out
-node_modules
\ No newline at end of file
+node_modules
+.vscode/*
```

Comment:
Why did you added a `.vscode` directory to git ignore?

Reply:
Not necessary but I figured it wasn't necessary, I can remove it if you want.

Reply:
We need `.vscode` directory to store our tasks for build and debug, so we have to keep it in repo. Let's remove `.vscode` from git ignore.